### DISCUSSION TITLE
Handeling dat at user

### DISCUSSION THREAD

**Discussion by Jee-Bee** [grd/FreePDM#57] [General]
This discussion point is also for memory:
what data do we use on the user site. 
- Is that user (CAD)data local or 
- Is the data a user table in the general database that is called when a change is requested (or on some refresh moments)

In the latter case i have really no idea how to handle it (for now)

**Comment by grd** [comment]
To answer your question, I don't see any difference between the two, local or a network, as long as you have a root directory for the data. When you check in the files, then the information should be gathered, not sooner, not later.

What do you mean with that you have no idea how to handle it? What problems are you facing?


(Please have a look at how Solid Edge is doing that (it's pretty good!). Do you have Windows? Then you can download the Community Edition, it's without any cost, at least that is what they tell us).

https://www.youtube.com/watch?v=IILyrMwDFfU
